src/App.js

```diff
@@ -30,40 +30,64 @@ class App extends Component {
   }
 
   async componentDidMount() {
+    this.setCurrencies()
+    this.checkToken();
+    this.addToFavorites();
+    this.addToNotes();
+  }
+
+  setCurrencies = async () => {
     const abbrevCurrencies = await this.cleaner.getAbbrevCurrencies();
     const expandedCurrencies = await this.cleaner.getExpandedCurrencies();
-    this.checkToken();
-    // this.addToFavorites();
-    // this.addToNotes();
     this.setState({
       abbrevCurrencies,
       expandedCurrencies
     });
   }
 
   addToFavorites = async () => {
-    const favorites = await fetchFavorites(this.state.token);
-    this.setState({ favorites });
+    let favorites
+    if (!this.state.token) {
+      favorites = [{
+        id: 12345,
+        name: "No favorites saved",
+        price_usd: 0,
+        percent_change_24_hr: 0
+      }]
+    } else {
```

Comment:
This was crucial. I'm thinking this will remove the 'No favorites saved' option once the favorites are imported now